tests/src/runner/podman-desktop-runner.ts

```diff
@@ -126,6 +124,21 @@ export class PodmanDesktopRunner {
         },
       },
     };
+    let args = {};
+    if (process.env.PODMAN_DESKTOP_BINARY) {
+      args = {
+        executablePath: process.env.PODMAN_DESKTOP_BINARY,
+      };
+    } else {
+      args = {
+        args: ['.'],
+      };
+    }
+    return {
+      ...args,
+      env,
+      ...recorder,
+    };
```

Comment:
might be easier to have args being undefined or set to [ '.'], and executablePath being also a variable, and return

`{ args, executablePath, env, etc}` instead of having args containing itself an args property